app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  has_many :likes
+  has_one :user
+
```

Comment:
- [ ] Please update the associations as below because a comment belongs to a user and a post.
 
 ```ruby
 belongs_to :author, class_name: 'User'
 belongs_to :post